crypto/x509/v3_addr.c

```diff
@@ -692,11 +692,20 @@ static int IPAddressFamily_cmp(const IPAddressFamily *const *a_,
     const ASN1_OCTET_STRING *a = (*a_)->addressFamily;
     const ASN1_OCTET_STRING *b = (*b_)->addressFamily;
     int len = ((a->length <= b->length) ? a->length : b->length);
+
```

Comment:
Nit: Please drop this empty line

Reply:
Ouch! You've removed the other empty line. You should've removed the empty line **above** int cmp declaration not below.